test/js/eventemitter.js

```diff
@@ -1,17 +1,23 @@
-const expect = require('chai').expect
+'use strict'
+/* global describe */
+/* eslint no-undef: "off" */
+/* global it */
+/* eslint no-undef: "off" */
```

Comment:
this is doubled up; you can say `/* eslint disable no-undef */` also, though it may make more sense to do this  per line since this actually tends to catch a lot of errors. Your call there on style